keras_contrib/applications/densenet.py

```diff
@@ -375,15 +376,16 @@ def __dense_block(x, nb_layers, nb_filter, growth_rate, bottleneck=False, dropou
         x = __conv_block(x, growth_rate, bottleneck, dropout_rate, weight_decay)
         x_list.append(x)
 
-        x = merge(x_list, mode='concat', concat_axis=concat_axis)
+        x1 = concatenate(x_list, axis=concat_axis)
```

Comment:
Not good! then all the convolutions will be the same! it is essential that all the concatenation be on x.